README.md

```diff
@@ -47,14 +47,153 @@ The following requirements are necessary to run the Bold BI solution.
 * **Web Browser:** Microsoft Edge | Mozilla Firefox | Chrome
 * **Database:** PostgreSQL 13.0+ | Microsoft SQL Server 2016+ | MySQL 8.0+ | Oracle Database 19c+
 
-## Deployment Methods
+## Quick Installation Guide for Bold BI
 
-There are two ways to deploy Bold BI on the Kubernetes cluster. Please refer to the following documents for Bold BI deployment:
+Use the following steps to quickly set up and deploy Bold BI.
 
-* [Deploy Bold BI using Kubectl](docs/index.md)
-* Deploy using Helm
-    1. [Deploy Bold BI using Helm](helm/README.md)
-    2. [Common Deployment(BI and Reports) using Helm](helm/bold-common/README.md)
+### 1. Ingress Controller Setup
+
+**Install NGINX Ingress Controller**
+
+   Follow official docs: [NGINX Ingress Deploy](https://kubernetes.github.io/ingress-nginx/deploy/)
+
+   Or install via Helm:
+
+   ```bash
+   helm repo add nginx-stable https://helm.nginx.com/stable
+
+   helm repo update
+   
+   helm install ingress-nginx ingress-nginx/ingress-nginx --version 4.13.1 --namespace ingress-nginx --create-namespace --set controller.service.externalTrafficPolicy=Local
+   ```
+
+  Refer to the Helm chart release details here: https://github.com/kubernetes/ingress-nginx
+
+**Note**:
+
+  * If you are using a domain name, make sure the Ingress Controller’s External IP is mapped to your DNS.
+
+  * If you are accessing Bold BI via the IP address directly, no DNS mapping is required.
+
+### 2. Add Bold BI Helm Repository
+
+```bash
+helm repo add boldbi https://boldbi.github.io/boldbi-kubernetes
+helm repo update
+helm search repo boldbi
+```
+
+Example output:
+
+```
+NAME            CHART VERSION   APP VERSION   DESCRIPTION
+boldbi/boldbi   13.1.10         13.1.10       Embed powerful analytics inside your apps
+```
+
+### 3. Deploy Bold BI
+
+#### Step 1: Create Namespace
+
+```bash
+kubectl create namespace [namespace-name]
+```
+
+#### Step 2: Install Bold BI
+
+General command:
+
+```bash
+helm install [RELEASE_NAME] boldbi/boldbi -n [namespace-name] \
+  --set appBaseUrl="https://<your-domain-or-ip>" \
+  --set clusterProvider="<aks/eks/gke/oke>" \
+  --set persistentVolume.<provider>.nfs.fileShareName="<storage-account>/<fileshare-name>" \
+  --set persistentVolume.<provider>.nfs.hostName="<storage-account>.file.core.windows.net"
+```
```

Comment:
@Priyasunil26 ,
You mentioned it as a general command, but the command seems very specific to aks.